kaptan/__init__.py

```diff
@@ -52,20 +53,39 @@ def upsert(self, key, value):
         self.configuration_data.update({key: value})
         return self
 
+    def _is_python_file(self, value):
```

Comment:
For consistency (and keeping with the pattern we want), we probably would want to keep "config type" (e.g. python, ini, json, yaml)-specific code in it's corresponding module.

For example:
We could create an 'extensions' property in each subclass of ``BaseHandler`` and check to see if that filetype is found. If so, pick that filetype's handler.

```python
class FileHandler(BaseHandler):

    def load(self, file_):
```
could have

```python
class FileHandler(BaseHandler):
    extensions = ['py']
    def load(self, file_):
```